src/OrchardCore.Modules/OrchardCore.Localization/Services/LocalizationService.cs

```diff
@@ -47,6 +53,16 @@ public async Task<string[]> GetSupportedCulturesAsync()
             ;
     }
 
+    /// <inheritdocs />
+    public IEnumerable<CultureInfo> GetAllCulturesAndAliases()
+    {
+        var cultures = CultureInfo.GetCultures(CultureTypes.AllCultures)
+            .Union(_cultureAliases)
+            .OrderBy(c => c.Name);
+
+        return cultures.ToArray();
```

Comment:
```suggestion
        return cultures;
```

It's not required to create an array here.